Makefile

```diff
@@ -272,7 +280,7 @@ $(tests) $(examples) : $(libceed)
 $(tests) $(examples) : LDFLAGS += -Wl,-rpath,$(abspath $(LIBDIR)) -L$(LIBDIR)
 
 run-% : $(OBJDIR)/%
-	@tests/tap.sh $(<:build/%=%)
+	#@tests/tap.sh $(<:build/%=%)
```

Comment:
Was this comment intentional?

Reply:
It comes from a PR I merged. @jeremylt I imagine this comment should not be merged?

Reply:
Correct, that shouldn't be there. I think it came from the CUDA branch.